src/protocol/libp2p/kademlia/store.rs

```diff
@@ -25,6 +25,9 @@ use crate::protocol::libp2p::kademlia::record::{Key, Record};
 
 use std::collections::{hash_map::Entry, HashMap};
 
+/// Logging target for the file.
+const LOG_TARGET: &str = "litep2p::ipfs::store";
```

Comment:
dq: Are we using ipfs in other places? Does it make sens for us to use `kad` here instead?

Reply:
Yes, we are using `litep2p::ipfs::kademlia` in other places. Corrected.